nevow/test/test_appserver.py

```diff
@@ -292,3 +330,145 @@ def test_emptyPostPath(self):
             (childResource, ()), request, ('foo', 'bar'), context)
         self.assertEqual(request.prepath, [''])
         self.assertEqual(request.postpath, [])
+
+
+class OldResourceAdapterTests(TestCase):
```

Comment:
More saliently though, I loved the docstrings on the other tests, could you add some for these as well?